x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -214,90 +175,73 @@ export const ActionTypeForm = ({
     <>
       {actionGroups && selectedActionGroup && setActionGroupIdByIndex && (
         <>
-          <EuiFlexGroup component="div">
-            <EuiFlexItem grow={true}>
-              <EuiFormControlLayout
-                fullWidth
-                prepend={
-                  <EuiFormLabel
-                    htmlFor={`addNewActionConnectorActionGroup-${actionItem.actionTypeId}`}
-                  >
-                    <FormattedMessage
-                      id="xpack.triggersActionsUI.sections.actionTypeForm.actionRunWhenInActionGroup"
-                      defaultMessage="Run when"
-                    />
-                  </EuiFormLabel>
-                }
-              >
-                <EuiSuperSelect
-                  fullWidth
-                  id={`addNewActionConnectorActionGroup-${actionItem.actionTypeId}`}
-                  data-test-subj={`addNewActionConnectorActionGroup-${index}`}
-                  options={actionGroups.map(({ id: value, name }) => ({
-                    value,
-                    inputDisplay: actionGroupDisplay(value, name, actionItem.actionTypeId),
-                    disabled: isActionGroupDisabled(value, actionItem.actionTypeId),
-                    'data-test-subj': `addNewActionConnectorActionGroup-${index}-option-${value}`,
-                  }))}
-                  valueOfSelected={selectedActionGroup.id}
-                  onChange={(group) => {
-                    setActionGroupIdByIndex(group, index);
-                    setActionGroup(group);
-                  }}
+          <EuiSuperSelect
+            prepend={
+              <EuiFormLabel htmlFor={`addNewActionConnectorActionGroup-${actionItem.actionTypeId}`}>
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.actionTypeForm.actionRunWhenInActionGroup"
+                  defaultMessage="Run when"
                 />
-              </EuiFormControlLayout>
-            </EuiFlexItem>
-          </EuiFlexGroup>
+              </EuiFormLabel>
+            }
+            fullWidth
+            id={`addNewActionConnectorActionGroup-${actionItem.actionTypeId}`}
+            data-test-subj={`addNewActionConnectorActionGroup-${index}`}
+            options={actionGroups.map(({ id: value, name }) => ({
+              value,
+              inputDisplay: actionGroupDisplay(value, name, actionItem.actionTypeId),
+              disabled: isActionGroupDisabled(value, actionItem.actionTypeId),
+              'data-test-subj': `addNewActionConnectorActionGroup-${index}-option-${value}`,
+            }))}
+            valueOfSelected={selectedActionGroup.id}
+            onChange={(group) => {
+              setActionGroupIdByIndex(group, index);
+              setActionGroup(group);
+            }}
+          />
+
           <EuiSpacer size="l" />
         </>
       )}
-      <EuiFlexGroup component="div">
```

Comment:
Same thing here, we didn't need the group and item because there was only one of them and they're only needed when aligning things horizontally.

This also addresses the overflow of text for the ComboBox and super selector that we were seeing.